networkx/algorithms/traversal/tests/test_bfs.py

```diff
@@ -51,6 +51,18 @@ def test_bfs_tree_isolates(self):
         assert sorted(T.nodes()) == [1]
         assert sorted(T.edges()) == []
 
+    def test_bfs_layers(self):
+        assert dict(enumerate(nx.bfs_layers(self.G, [0]))) == {
+            0: [0],
+            1: [1],
+            2: [2, 3],
+            3: [4],
+        }
```

Comment:
You should set this up to test both `sources=[0]` and `sources=0`.
That makes it worthwhile to create a variable to hold the expected values and the formatting is nicer:
```python
    expected = {0: [0], 1: [1], .....}
    assert dict(enumerate(nx.bfs_layers(self.G, [0]))) == expected
    assert dict(enumerate(nx.bfs_layers(self.G, 0))) == expected
```